docs/tutorials/02-local-simple-blueprint.md

```diff
@@ -0,0 +1,438 @@
+This simplifies the build and transport of the component as only one artifact has to be maintained.
+
+This tutorial uses the same resources as created in the first tutorial but now the artifacts are completely self-contained.
+For detailed information see [the blueprint-installation-relationship](../concepts/InstallationBlueprintRelationship.md).
+
+__Prerequisites__:
+- Helm commandline tool (see https://helm.sh/docs/intro/install/)
+- Component-cli (see https://github.com/gardener/component-cli)
+- [OPTIONAL] OCI compatible oci registry (e.g. GCR or Harbor)
+- Kubernetes Cluster (better use two different clusters: one for the landscaper and one for the installation)
+
+All example resources can be found in [./resources/local-ingress-nginx](./resources/local-ingress-nginx).<br>
+:warning: note that the repository `eu.gcr.io/gardener-project/landscaper/tutorials` is an example repository 
+and has to be replaced with your own registry if you want to upload your own artifacts.
+Although the artifacts are public readable so they can be used out-of-the-box without a need for your own oci registry.
+
+Structure:
+- [Resources](#resources)
+    - [Prepare nginx helm chart](#prepare-nginx-helm-chart)
+    - [Define Component Descriptor](#define-the-component-descriptor)
+    - [Create Blueprint](#create-blueprint)
+    - [Render and Validate](#render-and-validate-locally)
+- [Remote Upload](#remote-upload)
+- [Installation](#installation)
+- [Summary](#summary)
+- [Up next](#up-next)
+
+### Resources
+
+#### Define the Component Descriptor
+
+A component descriptor contains all resources that are used by the application installation.
+Resources are in this example the ingress-nginx helm chart but could also be `oci images` or even `node modules`.
+
+For more information about the component descriptor and the usage of the different fields see the [component descriptor docs](https://github.com/gardener/component-spec).
+
+```yaml
+meta:
+  schemaVersion: v2
+
+component:
+  name: github.com/gardener/landscaper/nginx-ingress
+  version: v0.2.0
+
+  provider: internal
+  sources: []
+  componentReferences: []
+  resources: []
+```
+
+#### Add local resources to the component descriptor
+
+The basic component ist defined, so now the artifacts that are needed for the deployment (helm chart and blueprints) are added to the component descriptor and uploaded to the oci registry.
+
+Local resources can be added to the component descriptor by using the `component-cli`.
+
+First the resources are defined in a `resources.yaml` configuration file.
+This configuration file contains 2 resource definition with the input to the previously created helm chart and blueprint.<br>
+:warning: Note that the paths used in this configuration file are relative to the location of the `resource.yaml`-file.
+```yaml
+---
+type: blueprint
+name: ingress-nginx-blueprint
+version: v0.1.0
+relation: local
+input:
+  type: "dir"
+  path: ./blueprint
+...
```

Comment:
what would be the meaning / intention of those `...`? Is there something else to put in here?

Reply:
they are part of the yaml spec and are the beginning `---` and end `...` characters for multi document yaml.
Most times people only use `---` but from the spec it should be `--- <some text> ...`.
See `9.2` fom the yaml spec https://yaml.org/spec/1.2/spec.html#id2800132

Maybe I can link that spec

Reply:
Linking the spec would be helpful. If you aren't aware of it, it just reads as if the snippet has been shortened.